sentry_sdk/integrations/aiohttp.py

```diff
@@ -286,7 +353,60 @@ def aiohttp_processor(
 
         return event
 
-    return aiohttp_processor
+    return aiohttp_server_processor
+
+
+def _make_client_processor(trace_config_ctx, response, response_content):
+    # type: (SimpleNamespace, Response, Optional[Dict[str, Any]]) -> EventProcessor
+    def aiohttp_client_processor(
+        event,  # type: Dict[str, Any]
+        hint,  # type: Dict[str, Tuple[type, BaseException, Any]]
+    ):
+        # type: (...) -> Dict[str, Any]
+        with capture_internal_exceptions():
+            request_info = event.setdefault("request", {})
+
+            parsed_url = parse_url(str(response.url), sanitize=False)
+            request_info["url"] = parsed_url.url
+            request_info["query_string"] = parsed_url.query
+            request_info["method"] = response.method
+
+            if getattr(trace_config_ctx, "request_headers", None):
+                request_info["headers"] = _filter_headers(
+                    dict(trace_config_ctx.request_headers)
+                )
+            if getattr(trace_config_ctx, "request_body", None):
+                request_info["data"] = trace_config_ctx.request_body
+
+            if response.url.path == "/graphql":
+                request_info["api_target"] = "graphql"
+
+                query = request_info.get("data")
+                if response.method == "GET":
+                    query = dict(parse_qsl(parsed_url.query))
+
+                if query:
+                    operation_name = _get_graphql_operation_name(query)
+                    operation_type = _get_graphql_operation_type(query)
+                    event["fingerprint"] = [
+                        operation_name,
+                        operation_type,
+                        response.status,
+                    ]
+                    event["exception"]["values"][0][
+                        "value"
+                    ] = "GraphQL request failed, name: {}, type: {}".format(
+                        operation_name, operation_type
+                    )
+
+                if response_content:
+                    contexts = event.setdefault("contexts", {})
+                    response_context = contexts.setdefault("response", {})
+                    response_context["data"] = response_content
```

Comment:
Guard with with `_should_send_default_pii()`. Or is the graphql error context just error messages?

Reply:
Could still contain data afaik. Adding the guard.